gradle/dependencies.gradle

```diff
@@ -166,6 +166,7 @@ libs += [
   kafkaStreams_23: "org.apache.kafka:kafka-streams:$versions.kafka_23",
   kafkaStreams_24: "org.apache.kafka:kafka-streams:$versions.kafka_24",
   kafkaStreams_25: "org.apache.kafka:kafka-streams:$versions.kafka_25",
+  kafkaStreams_26: "org.apache.kafka:kafka-streams:$versions.kafka_26",
```

Comment:
I'm not 100% sure if this is actually supposed to be on the 2.6 branch or not, but the `kafka_26` was already defined and it seems odd to have one but not the other. If adding this is wrong, I can take out the `kafka_26` instead 🤷‍♀️

Reply:
We don't need this for `2.6` branch, as when we test upgrade from older version to `2.6` we just use the jar we just built base on `2.6` branch -- we only need to pull in older versions.

Reply:
So I can/should remove the `kafka_26` from this branch as well, yes?

Reply:
Well, this PR adds `kafka_26` but we don't need to add it -- not sure what you want to remove?

Reply:
not ` kafkaStreams_26`, `kafka_26` (defined above) -- it just seems weird to me that we would define one but not the other? But I guess it doesn't hurt so I can close this PR

Reply:
Ah. Yes. Could be removed. It's unused. But also does not matter to have an unused var defined...

Reply:
Seems this PR added if for no reason (but also no hard): https://github.com/apache/kafka/commit/9ebc71e3d7d1e361b3844d907c4ae935fcda1240